pkg/cli/admin/upgrade/status/examples/4.15.0-ec2-unavailable-mco-20m.detailed-output

```diff
@@ -25,6 +25,15 @@ ip-10-0-4-159.us-east-2.compute.internal    Outdated     Pending   4.14.0-rc.3
 ip-10-0-99-40.us-east-2.compute.internal    Outdated     Pending   4.14.0-rc.3   ?     
 
 = Update Health =
+Message: Alert: Target disappeared from Prometheus target discovery.
```

Comment:
This one is `pending` in your `4.15.0-ec2-unavailable-mco-20m-alerts.json`.  We probably want to filter to just `.State == "firing"`?